### ISSUE TITLE
SFX

### ISSUE THREAD

**Issue by panteroni** [MA-GD21/toobin#8]
SFX that are already done: Collision with obstacles, going through gates, throwing a can, water splash, rotating, animals/enemies, hitting the gate, letters at the end of the game (on + off)
hitting the enemy with a can.

More SFX can be made once we're clear with all the events that could happen in our game.

**Comment by Aadjou** [comment]
- For now no more sound effects are needed (there is 11 now!)
- evtl. we could do some refinement to make them more unified / 8bit
- get some help from @andibissig to discuss potentials
- make refinements only after we hear the sounds play out in the actual game
